### PR TITLE
Extend compact format

### PR DESCRIPTION
**Pull request by thomas-zahner**
Addresses https://github.com/lycheeverse/lychee/issues/1477

### CODE REVIEW

File: lychee-bin/src/formatters/stats/compact.rs [hunk 1]
```diff
@@ -51,25 +53,40 @@ impl Display for CompactResponseStats {
             writeln!(f)?;
         }
 
-        color!(f, NORMAL, "\u{1F50D} {} Total", stats.total)?;
+        color!(f, NORMAL, "🔍 {} Total", stats.total)?;
```
Comment (thomas-zahner) [inline lychee-bin/src/formatters/stats/compact.rs:56]: @mre I'd personally prefer this inlined version as it is much more readable compared to the code and a comment at the end of the line (if any). Was there a reason you don't use the emoji directly?

Comment (mre) [reply lychee-bin/src/formatters/stats/compact.rs:56]: Talked to @m3t0r and we both don't see a reason why using the emoji directly is an issue. So, sure, let's change it back. Easier to read anyway.

**Comment by mre** [comment]
Thanks! 🥳

File: lychee-bin/src/formatters/stats/compact.rs (hunk 1 above)
Comment (thomas-zahner) [reply lychee-bin/src/formatters/stats/compact.rs:56]: Sounds great :+1:

**Pull request merged**